ssg/build_profile.py

```diff
@@ -51,15 +51,17 @@ def __init__(self, filepath):
             if rule_id is None:
                 raise RuntimeError("Can't index a rule with no id attribute!")
 
-            assert(rule_id not in self.indexed_rules)
+            if rule_id in self.indexed_rules:
+                raise RuntimeError("Rule exists with multiple id attributes: %s!" % rule_id)
```

Comment:
shouldn't this be "multiple rules with the same id"?

Reply:
@mpreisler Updated; good catch